lib/grinder_files.dart

```diff
@@ -386,16 +386,13 @@ void _copyImpl(FileSystemEntity entity, Directory destDir, [GrinderContext conte
 /**
  * Delete the given file entity reference.
  */
-void delete(FileSystemEntity entity, [GrinderContext context]) {
-  _deleteImpl(entity, context);
+void delete(FileSystemEntity entity, [GrinderContext _context]) {
```

Comment:
Since `delete` hasn't been released yet, we can remove the `_context` parameter right?